### DISCUSSION TITLE
Error with make command

### DISCUSSION THREAD

**Discussion by R1070V3RK1LL** [Almamu/linux-wallpaperengine#140] [Q&A]
Librairies were installed on my ubuntu-20.04, but there stil is an error:
![image](https://user-images.githubusercontent.com/60746919/215122163-b8aa2792-e583-4dac-8dac-0e8171fdbf79.png)

**Comment by Svallinn** [comment]
https://github.com/Almamu/linux-wallpaperengine/issues/121#issuecomment-1194911528

**Comment by Almamu** [comment]
I've pushed a commit that should fix the issue, as shown in #121 it was caused by an error in the FindFFMPEG.cmake used. With the last commit it should be fixed.

**Comment by R1070V3RK1LL** [comment]
Thank you very much, I will try again tomorrow!

**Comment by R1070V3RK1LL** [comment]
Update: solved the cmake command issue by installing manually cmake 3.18.6, but I had the SDL_mixer.h missing error (which is surprising because it does exist in usr/include/SDL).
![image](https://user-images.githubusercontent.com/60746919/216386395-0c5b0157-d566-43a2-a91f-560010d88ce1.png)

 Then another error happened after moving the SDL header files out of the SDL folder, at the end of the make command: 
![image](https://user-images.githubusercontent.com/60746919/216385928-cc39c7e2-7d4e-4b34-99c7-527b7bbfc494.png)

**Reply by Almamu** [reply]
I've updated all the SDL code to use SDL2 now and got rid of mixer. This should fix your issues with the compilation. Can you try with the latest commit I pushed?

**Reply by R1070V3RK1LL** [reply]
I tried again but I think there is a missing FindSDL2.cmake file inside CMakeModules because as SDL2 can't be found (after installing it), this error is displayed:
![image](https://user-images.githubusercontent.com/60746919/217757351-00440ce3-29cb-4797-9888-45bbd72a9617.png)

**Reply by Almamu** [reply]
The SDL2 config should be provided by libsdl2-dev package in ubuntu/debian, is that one installed?

**Reply by R1070V3RK1LL** [reply]
Thanks, however I still got the above errors after make command:  "usr/local/lib/libavcodec(tiff.o): undefined reference to symbol 'inflateEnd' " and "usr/lib/x86_64-linux-gpu/libz.so: error adding symbols:DSO missing from command line"
